js/front/dataset/index.js

```diff
@@ -141,6 +148,35 @@ new Vue({
         },
 
         /**
+         * Get a cached checked result from extras if fresh enough
+         * @param  {Object} resource A resource as extracted from JSON-LD
+         */
+        getCachedCheck(resource) {
+            const extras = resource.extras;
+            if (extras['check:date']) {
+                const checkDate = new Date(extras['check:date']);
+                if (checkDate >= this.limitCheckDate) {
+                    return Object.keys(extras).reduce(function(obj, key) {
```

Comment:
Arrow function for the win :)

Reply:
:+1:

Reply:
I don't think an arrow function is relevant here, it only adds the extra cost of scoping without being used

Reply:
Micro-perf vs. consistency

Reply:
These are not 2 equivalent syntaxes so consistency is not relevant. On the breaking cases (which we fixed by switching back on functions), what should we do ? Keep a bug for consistency ?
This is not at all about micro-perfs but about the fact that over-using arrow functions when it's not necessary introduced bugs before (and I'm pretty sure there still is) and so being mechanical about the use of it is not safe (mostly because these bugs are a pain to detect and so a big useless loss of time)